R/fitting-and-postprocessing.R

```diff
@@ -33,19 +20,11 @@ extract_lognormal_draws <- function(data, id_vars, from_dt = FALSE) {
   draws <- posterior::as_draws_df(draws) |>
     data.table::as.data.table()
 
-  data.table::setnames(
-    draws, c("Intercept", "Intercept_sigma"), c("meanlog", "sdlog_log"),
-    skip_absent = TRUE
-  )
-
-  data.table::setnames(
-    draws, c("b_Intercept", "b_sigma_Intercept"), c("meanlog", "sdlog_log"),
-    skip_absent = TRUE
-  )
-
-  draws <- draws[, sdlog := exp(sdlog_log)]
-  draws <- draws[, list(meanlog, sdlog)]
-  draws <- add_natural_scale_mean_sd(draws)
+  draws <- draws[, mu := Intercept]
```

Comment:
This is similar but not the same as the code above. This creates a new variable and leaves the old one. The old code got rid of the original variable name. Justify?

Reply:
aha yes